snetd/cmd/components.go

```diff
@@ -236,7 +255,7 @@ func (components *Components) FreeCallUserService() escrow.FreeCallUserService {
 
 	components.freeCallUserService = escrow.NewFreeCallUserService(
 		components.FreeCallUserStorage(),
-		escrow.NewEtcdLocker(components.AtomicStorage(), components.ServiceMetaData()),
+		escrow.NewEtcdLocker(components.AtomicStorage()),
```

Comment:
Free-call locks are not MPE specific:
```suggestion
		escrow.NewEtcdLocker(escrow.NewPrefixedAtomicStorage(components.AtomicStorage(), "/free-call/lock")),
```